reckless_cmd/src/reckless/cmd.rs

```diff
@@ -39,6 +39,6 @@ pub enum RecklessCommand {
 
 #[derive(Debug, Subcommand)]
 pub enum RemoteAction {
-    Add { name: String, url: String },
```

Comment:
The name is useful here because you may want assign your custom name to a repository instead of the name inside the URL.

This is useful I think